quarkc/command.py

```diff
@@ -183,21 +185,24 @@ def main(args):
 
         filenames = args["<file>"]
         for url in filenames:
+            c = compiler.Compiler()
+            c.version_warning = args["--version-warning"]
             if args["install"]:
-                compiler.install(url, *backends)
+                compiler.install(c, url, *backends)
             elif args["compile"]:
-                compiler.compile(url, output, *backends)
+                compiler.compile(c, url, output, *backends)
             elif args["run"]:
-                compiler.run(url, args["<args>"], *backends)
+                compiler.run(c, url, args["<args>"], *backends)
             elif args["docs"]:
-                compiler.make_docs(url, output)
+                compiler.make_docs(c, url, output)
             else:
                 assert False
     except (KeyboardInterrupt, QuarkError) as err:
         if not args["run"]:
             shell.command_log.error("")
         return err
     except:
+        raise
```

Comment:
@bozzzzo I was forced to re-rase the exception here, because otherwise during development some of my exceptions get swallowed. What should we do about it?